tests/unit/core/directives/ViewDirective.spec.js

```diff
@@ -12,7 +12,7 @@ describe('Action Dialog Plugin Testing', () => {
     name: 'SideDrawer',
     template:
       '<StackLayout class="sidedrawer"><StackLayout ref="drawerContent" class="drawer-content"></StackLayout>' +
-      '<StackLayout ref="mainContent" class="main-content"></StackLayout><slot /></StackLayout>',
```

Comment:
why did you deleted slot? is it related with sideDrawer?

Reply:
I was trying to figure out what it does, and actually it does not affect the test, but shall revert it to slotted version just in case.